css.php

```diff
@@ -15,26 +15,64 @@
 require_once "./inc/init.php";
 require_once MYBB_ROOT . $config['admin_dir'] . '/inc/functions_themes.php';
 
-$stylesheet = $mybb->get_input('stylesheet', MyBB::INPUT_INT);
+$stylesheets = $mybb->get_input('stylesheet', MyBB::INPUT_ARRAY);
```

Comment:
It looks like the `stylesheet=$sid` format might still exist in the database - it might be a good idea to convert these to `[$sid]` arrays.

Reply:
I'll have to have a look at the database for an install using `css.php` and see. I think that will be handled by the `global.php` changes though.